### PR TITLE
Add marker field to ONS dataset 1 and update column names

### PR DESCRIPTION
**Pull request by allait**
### Description of change

Marker is set to not-applicable to JE region records, which
don't have a value in the original dataset.

Updates column names to match other ONS datasets.

### Checklist

* [ ] Have tests been added to cover any changes?
* [ ] Has the [CHANGELOG](https://github.com/uktrade/data-flow/blob/master/CHANGELOG.md) been updated?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: dataflow/dags/ons_pipelines.py [hunk 1]
```diff
@@ -27,25 +28,29 @@ class ONSUKSATradeInGoodsPipeline(_ONSPipeline):
         table_name="ons_uk_sa_trade_in_goods",
```
Comment (samuelhwilliams) [inline dataflow/dags/ons_pipelines.py:28]:
```suggestion
        table_name="ons__uk_sa_trade_in_goods",
```

File: dataflow/dags/ons_pipelines.py [hunk 2]
```diff
@@ -27,25 +28,29 @@ class ONSUKSATradeInGoodsPipeline(_ONSPipeline):
...
         field_mapping=[
             (None, sa.Column("id", sa.Integer, primary_key=True, autoincrement=True)),
```
Comment (samuelhwilliams) [inline dataflow/dags/ons_pipelines.py:30]: Can we also add the id/import_time columns here that are in the other ONS datasets now?

File: dataflow/dags/ons_pipelines.py (hunk 1 above)
Comment (samuelhwilliams) [reply dataflow/dags/ons_pipelines.py:28]: And wherever this filters through into the SQL query...

File: dataflow/dags/ons_pipelines.py [hunk 3]
```diff
@@ -27,25 +28,29 @@ class ONSUKSATradeInGoodsPipeline(_ONSPipeline):
...
-            (("unit", "value"), sa.Column("unit", sa.String)),
+            (None, sa.Column("import_time", sa.DateTime, default=datetime.utcnow)),
+            (
+                ("geography_code", "value"),
+                sa.Column("og_ons_iso_alpha_2_code", sa.String),
+            ),
+            (("geography_name", "value"), sa.Column("og_ons_region_name", sa.String)),
+            (("period", "value"), sa.Column("og_period", sa.String)),
+            (("direction", "value"), sa.Column("og_direction", sa.String)),
+            (("total", "value"), sa.Column("og_total", sa.Numeric)),
```
Comment (samuelhwilliams) [inline dataflow/dags/ons_pipelines.py:39]: Does this come through from sparql as an integer or a string?

File: dataflow/dags/ons_pipelines.py [hunk 4]
```diff
@@ -27,25 +28,29 @@ class ONSUKSATradeInGoodsPipeline(_ONSPipeline):
...
-            (("direction", "value"), sa.Column("direction", sa.String)),
-            (("total", "value"), sa.Column("total", sa.Numeric)),
-            (("unit", "value"), sa.Column("unit", sa.String)),
+            (None, sa.Column("import_time", sa.DateTime, default=datetime.utcnow)),
+            (
+                ("geography_code", "value"),
+                sa.Column("og_ons_iso_alpha_2_code", sa.String),
+            ),
+            (("geography_name", "value"), sa.Column("og_ons_region_name", sa.String)),
+            (("period", "value"), sa.Column("og_period", sa.String)),
```
Comment (samuelhwilliams) [inline dataflow/dags/ons_pipelines.py:37]: Are values for period here something like `quarter/2010-Q11`?

File: dataflow/dags/ons_pipelines.py (hunk 2 above)
Comment (allait) [reply dataflow/dags/ons_pipelines.py]: I've added import time, changed this to UUID now

File: dataflow/dags/ons_pipelines.py (hunk 3 above)
Comment (allait) [reply dataflow/dags/ons_pipelines.py:46]: Everything comes in as a string

File: dataflow/dags/ons_pipelines.py (hunk 4 above)
Comment (allait) [reply dataflow/dags/ons_pipelines.py:44]: "2008" or "2008-01"

Comment (allait) [reply dataflow/dags/ons_pipelines.py:44]: There are no quarters in this dataset, only months and years

File: dataflow/dags/ons_pipelines.py (hunk 1 above)
Comment (allait) [reply dataflow/dags/ons_pipelines.py]: Updated

File: dataflow/dags/ons_pipelines.py (hunk 4 above)
Comment (samuelhwilliams) [reply dataflow/dags/ons_pipelines.py:44]: Ah ok fair enough. Can we add a `norm_period_type` for `year`/`month` just derived from the length of those? I know it'll be asked for.

File: dataflow/dags/ons_pipelines.py (hunk 3 above)
Comment (samuelhwilliams) [reply dataflow/dags/ons_pipelines.py:46]: Ok - can we take the string total in to a string column `og_total` and then have a numeric column for `norm_total` please?

Comment (allait) [reply dataflow/dags/ons_pipelines.py:46]: Um, sure. They'll be exactly the same but 🤷

File: dataflow/dags/ons_pipelines.py (hunk 4 above)
Comment (allait) [reply dataflow/dags/ons_pipelines.py:52]: Added

File: dataflow/dags/csv_pipelines/ons_csv_pipelines.py [hunk 5]
```diff
@@ -28,124 +28,115 @@ class ONSUKSATradeInGoodsCSV(_CSVPipelineDAG):
...
-                ELSE 'month'
-            END AS period_type,
-            'total trade' as direction,
-            e.total + i.total as trade_value,
-            i.unit,
+            i.og_ons_iso_alpha_2_code as ons_geography_code,
+            i.og_ons_region_name as geography,
+            i.og_period,
+            i.norm_period_type,
+            'norm_total trade' as og_direction,
```
Comment (samuelhwilliams) [inline dataflow/dags/csv_pipelines/ons_csv_pipelines.py:62]: Oops :)

Comment (allait) [reply dataflow/dags/csv_pipelines/ons_csv_pipelines.py:62]: oh, nice catch! Search and replace gone wrong 😅 fixed

Comment (samuelhwilliams) [reply dataflow/dags/csv_pipelines/ons_csv_pipelines.py:62]: Yeah I did exactly the same thing myself...

Comment (samuelhwilliams) [reply dataflow/dags/csv_pipelines/ons_csv_pipelines.py:62]: I think you actually missed this one and fixed an additional instance of this on line 124ish 😅

Comment (allait) [reply dataflow/dags/csv_pipelines/ons_csv_pipelines.py]: 🤦‍♂️

**Pull request merged**